bravado/http_client.py

```diff
@@ -197,6 +197,9 @@ def __init__(self, session, request):
         self.session = session
         self.request = request
 
+    def result(self, *args, **kwargs):
```

Comment:
Something like that, we should be able to delete `handle_response` from the module, so it would probably be more like:

```python
return self.swagger_spec.http_client.request(request, self)
```

which lets us remove the `response_future` as well (it's just a closure so we have access to the operation anyway).